exercisesSubmission.json

```diff
@@ -1,18 +1,18 @@
 {
-  "day": "day-18",
-  "segmentName": "segment-1",
-  "exercises": {
-    "express-101": {
-      "completed": false
-    },
-    "templating-engine": {
-      "completed": false
-    },
-    "express-forms": {
-      "completed": false
-    },
-    "express-router": {
-      "completed": false
+    "day": "day-18",
+    "segmentName": "segment-1",
+    "exercises": {
+        "express-101": {
+            "completed": true
+        },
+        "templating-engine": {
+            "completed": false
+        },
+        "express-forms": {
+            "completed": true
+        },
+        "express-router": {
+            "completed": false
+        }
     }
-  }
 }
```

Comment:
Should you end file without a new line? 
Github is suggesting to have new line at the end of each file. What do you think? I think it is good idea to always have new line.
There is one article explaining the issue here: https://evanhahn.com/newline-necessary-at-the-end-of-javascript-files/

Reply:
Thanks for the link, I do agree on this.

Reply:
I've rectified the PR and am resolving this now.